Purchases/Purchasing/Purchases.swift

```diff
@@ -194,6 +194,8 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
     /// Current version of the Purchases framework.
     @objc public static var frameworkVersion: String { SystemInfo.frameworkVersion }
 
+    /// Whether transactions should be finished automatically.
```

Comment:
I don't love this one. Do we have a link or something that we can link to with better docs?

Reply:
I don't think we have solid docs on this, since we don't want people using it unless they really know what they're doing. 

How about: 


```swift
/** Whether transactions should be finished automatically.
 * `true` by default.
 * - Warning: Setting this value to `false` will prevent the SDK from finishing transactions.
 * In this case, you *must* finish transactions in your app, otherwise they will remain in the queue and
 * will turn up every time the app is opened.
 * More information on finishing transactions manually [is available here](https://developer.apple.com/documentation/storekit/original_api_for_in-app_purchase/finishing_a_transaction)
```

We can create a rev.cat link for that one so we don't point directly to Apple's docs.

Reply:
Maybe `- Important:` is better?
here's a comparison:
| Important | Warning |
| :-: | :-: |
| <img width="788" alt="image" src="https://user-images.githubusercontent.com/3922667/151811396-684e53f8-78c8-4ab8-a2c5-8bb5042373c3.png"> | <img width="808" alt="image" src="https://user-images.githubusercontent.com/3922667/151811343-c5e3774e-4f36-4d16-8876-26a1b535b22e.png"> |

Reply:
well, nevermind 😂 from the [docs](https://www.swift.org/documentation/docc/formatting-your-documentation-content#see-also), I think `Warning` might be more appropriate
<img width="825" alt="image" src="https://user-images.githubusercontent.com/3922667/151811512-2c26c1cb-7753-4099-ae7c-4248b276dc85.png">

Reply:
Thanks! This looks great.
I don't have access to create a `rev.cat` link though.